python-black.sublime-settings

```diff
@@ -8,6 +8,8 @@
    //    - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
    "format_on_save": "on",
    // Black [OPTIONS]
+   // The priority of loading options for Black is:
+   // Sublime project settings > Configuration file > Sublime package user settings > Sublime package default settings
```

Comment:
I think `sublime project settings` should only be used if `pyproject.toml` does not exist in the project root directory.

I think a better priority should be:

Project Configuration file > Sublime Project Settings > Black Global Configuration File > Sublime Package User Settings > Sublime Package Default Settings = Black Default Settings

Reply:
In my opinion, that priority doesn't work for current version because the way to fine the configuration files is using the 
`find_pyproject_toml` in black lib and it also finds the global configuration file if exists.
So if there's a configuration file exists either in project or global path, the sublime project settings will never affect to them. 
https://github.com/thep0y/python-black/blob/7bbc32129de95bfc31d02d8523968a012e7c5cdd/python_black/lib/black/files.py#L78-L95

I think only it's necessary, the developer will use the sublime project settings to overwrite configs without modifying any configuration files, and in general circumstances, the IDE's project files also won't be included to VCS. 

In briefly. sublime project settings is useful when the developers want to ignore the configuration files, and that's why I let it to be the highest priority.

Reply:
Ok, since this is not a feature I need, it may be removed or modified if it bothers other users in the future.

Reply:
Ok, thanks for your understanding.